packages/frontend/.cct/utils/get-template-component-webpack-config.js

```diff
@@ -111,18 +104,26 @@ export const getTemplateComponentWebpackConfig = (
             // TODO: further globals might be passed by carrot-scripts??
             new webpack.DefinePlugin({
                 ...globals,
-                __DEV__: devMode ? JSON.stringify(true) : JSON.stringify(false),
+                ROOT_ID: JSON.stringify(POSTCSS_PREFIX),
```

Comment:
Should we keep the leading&tailing `__` for the `ROOT_ID` name? Like `__DEV__`.

Reply:
Fixed in d401af4d5f4912dbe611cc8c6873818512450f0a